tst/EnergyPlus/unit/MixedAir.unit.cc

```diff
@@ -219,6 +219,29 @@ namespace EnergyPlus {
 			"    ,                        !- High Humidity Outdoor Air Flow Ratio",
 			"    Yes,                     !- Control High Indoor Humidity Based on Outdoor Humidity Ratio",
 			"    BypassWhenWithinEconomizerLimits;  !- Heat Recovery Bypass Control Type", // HX bypass should be true
+
+			"  OutdoorAir:Mixer,",
+			"    OA Mixer 1,                !- Name",
+			"    Mixed Air Node 1,          !- Mixed Air Node Name",
+			"    Outside Air Inlet Node 1, !- Outdoor Air Stream Node Name",
+			"    Relief Air Outlet Node 1,  !- Relief Air Stream Node Name",
+			"    VAV Sys 1 Inlet Node;     !- Return Air Stream Node Name",
+
+			" AirLoopHVAC:ControllerList,",
+			"    OA Sys 1 controller,     !- Name",
+			"    Controller:OutdoorAir,   !- Controller 1 Object Type",
+			"    OA Controller 1;         !- Controller 1 Name",
+
+			" AirLoopHVAC:OutdoorAirSystem:EquipmentList,",
+			"    OA Sys 1 Equipment list, !- Name",
+			"    OutdoorAir:Mixer,        !- Component 1 Object Type",
+			"    OA Mixer 1;                !- Component 1 Name",
+
+			" AirLoopHVAC:OutdoorAirSystem,",
+			"    OA Sys 1, !- Name",
+			"    OA Sys 1 controller,     !- Controller List Name",
+			"    OA Sys 1 Equipment list; !- Outdoor Air Equipment List Name",
+
```

Comment:
The new method actually simulates the OA sys components so a little more info is needed for the unit test.